File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.cpp
class Solution {
public:
    int maxProfit(vector<int>& prices) {
        int buyMin{prices[0]};
        int res{};
        for (int i{1}; i < prices.size(); i++) {
            buyMin = min(buyMin, prices[i-1]);
            res = max(prices[i] - buyMin, res);
             
        }
        return res; 
        
        
    }
};

File: Data Structures & Algorithms/maximum-number-of-balloons/submission-0.py
class Solution:
    def maxNumberOfBalloons(self, text: str) -> int:
        b_cnt = Counter("balloon")


        cnt = Counter(text)

        res = []


        for c in b_cnt:
            res.append(cnt[c] // b_cnt[c])
        return min(res)
            
            
